src/Mod/Draft/draftguitools/gui_groups.py

```diff
@@ -394,3 +416,56 @@ def Activated(self):
 
 Draft_AddConstruction = AddToConstruction
 Gui.addCommand('Draft_AddConstruction', AddToConstruction())
+
+class AddNamedGroup(gui_base.GuiCommandSimplest):
+
+    """Gui Command for the addGroup tool.
+        It adds a new named group
+    """
+    def __init__(self):
+        super().__init__(name=_tr("Add a new group with a given name"))
+
+
+    def GetResources(self):
+        """Set icon, menu and tooltip."""
+        _menu = "Add a new named group"
+        _tip = ("Add a new group with a given name.")
+
+        d = {'Pixmap': 'Draft_AddNamedGroup',
+             'MenuText': QT_TRANSLATE_NOOP("Draft_AddNamedGroup", _menu),
+             'ToolTip': QT_TRANSLATE_NOOP("Draft_AddNamedGroup", _tip)}
+        return d
+
+    def Activated(self):
+        super().Activated()
+        panel = Ui_AddNamedGroup()
+        Gui.Control.showDialog(panel)
+        panel.name.setFocus()
+
+
+Draft_AddNamedGroup = AddNamedGroup
+Gui.addCommand('Draft_AddNamedGroup', AddNamedGroup())
+
+
+class Ui_AddNamedGroup():
```

Comment:
Even if it's simple, this task panel should be defined in a module like `drafttaskpanels/view_addnamedgroup.py`. Ideally the interface is a .ui file, but in this case it's so small that maybe it doesn't make much sense. Still, we would like to structure the code into objects, Gui Commands, and then task panels.